src/controllers/postControllers.js

```diff
@@ -0,0 +1,27 @@
+import {postRepository} from '../repositories/index.js'
+ 
+
+async function timeline(req, res){
+    try{
+        const posts = await postRepository.getPost();
+        res.status(200).send(posts);
+    }
+    catch{
+        res.sendStatus(500);
+    }
+}
+
+async function sendPost(req, res){
+    const {content, postLink} = req.body;
+    const userId = res.locals.data.id;
+    try{
+        await postRepository.savePostInDatabase(userId,content,postLink);
+        res.sendStatus(201);
+    } 
+    catch(error){
+        console.log(error);
+        res.status(500).send("houve um erro ao armazernar o post");
+    }
+}
+
+export {timeline, sendPost};
```

Comment:
Personally i like to use the export keyword before every new function statement instead of exporting here. Its a way to not forget to export the function before using it elsewhere.